wallet/cardano-sl-wallet.cabal

```diff
@@ -172,9 +172,9 @@ library
                        -O2
 
   -- linker speed up for linux
-  if os(linux)
-    ghc-options:       -optl-fuse-ld=gold
-    ld-options:        -fuse-ld=gold
+  -- if os(linux)
```

Comment:
This seems like a transient comment you put to solve the linker shenanigans, right? I guess we should restore it.

Reply:
No, we shouldn't, and it's better actually to delete this instead of commenting out - GHC simply cannot link a dependency if it's linked with `gold`, and `cardano-sl-wallet` is a dependency of `cardano-sl-wallet-new`. That's why I had problems with linking. You didn't experience that probably because you use Mac or something.

Reply:
Ah, TIL!